docs/pages/web/datapoint.js

```diff
@@ -242,7 +242,6 @@ Be cautious of using Datapoint for large volumes of data. In cases where the con
         <MainSection.Subsection
           description={`The \`badge\` prop is available to add a [Badge](/web/badge) to the title. This feature is in beta, so please use cautiously.`}
```

Comment:
`This feature is in beta, so please use cautiously.`

Should this sentence also be removed if we're removing the "Beta" badge?

Reply:
DONE